templates/vue/src/components/TapestrySidebar.vue

```diff
@@ -179,6 +187,13 @@ export default {
         query: this.$route.query,
       })
     },
+    editNode() {
```

Comment:
Forgive my lack of knowledge here but is there a reason we have to create a new route? I'm wondering if there's any way to use the existing edit route logic.

Reply:
I'll take a look. I think we might be able to use mixins to avoid duplicating the call in TapestryNode or add something along the lines of this when mounting TapestryNode:
```
      this.$root.$on("edit-node", id => {
        this.$router.push({
          name: names.MODAL,
          params: { nodeId: id, type: "edit", tab: "content" },
          query: this.$route.query,
        })
      })
```

Reply:
@rlnsy Thoughts on the update?